nixos/tests/nexus.nix

```diff
@@ -6,29 +6,27 @@
 import ./make-test.nix ({ pkgs, ...} : {
```

Comment:
Please update/clarify the comments at the top of this file describing what this test does (github doesn't let me comment on lines not changed by this PR).